core/src/Flags.h

```diff
@@ -67,6 +67,7 @@ class Flags
 	int count() const noexcept { return BitHacks::CountBitsSet(i); }
 
 	constexpr inline bool operator==(Flags other) const noexcept { return i == other.i; }
+	constexpr inline bool operator!=(Flags other) const noexcept { return i != other.i; }
```

Comment:
What made this addition mandatory? The `Q_DECLARE_FLAGS(BarcodeFormats, BarcodeFormat)` below is completely independent of my `Flags` class, right?

Reply:
This operator is used at line 362 of ZXingQtReader.h.

`Q_DECLARE_FLAGS(BarcodeFormats, BarcodeFormat)` is indeed independent of `ZXing::Flags`, but `BarcodeReader` uses `ZXing::BarcodeFormats` and only converts to `ZXingQt::QML::BarcodeFormats` (= `QFlags`) when exposing it to QML.

Reply:
> This operator is used at line 362 of ZXingQtReader.h.

Ah, and that used to be an integer comparison...